timemachine/fe/atom_mapping.py

```diff
@@ -52,13 +52,19 @@ def compare(self, p, mol1, atom1, mol2, atom2):
             return True
 
 
-def mcs_map(a, b, threshold: float = 0.5, timeout: int = 5, smarts: Optional[str] = None):
+def mcs_map(a, b, threshold=2.0, timeout: int = 5, smarts: Optional[str] = None):
     """Find the MCS map of going from A to B"""
     params = rdFMCS.MCSParameters()
     params.BondCompareParameters.CompleteRingsOnly = 1
     params.BondCompareParameters.RingMatchesRingOnly = 1
-    params.AtomCompareParameters.matchValences = 1
-    params.AtomTyper = CompareDist(threshold=threshold)
+    params.BondTyper = rdFMCS.BondCompare.CompareAny
+
+    params.AtomCompareParameters.CompleteRingsOnly = 1
+    params.AtomCompareParameters.RingMatchesRingOnly = 1
+    params.AtomCompareParameters.matchValences = 0
+    params.AtomCompareParameters.MaxDistance = threshold
+    params.AtomTyper = rdFMCS.AtomCompare.CompareAny
```

Comment:
Commenting on some of these settings might be useful (esp. those that changed). E.g., why did we change `matchValences` to 0?

Reply:
matchValences was unnecessary (and was a default from an older protocol that tried to avoid chiral additions, eg. amides into tetrahedral centers). Chiral restraints should naturally deal with this issue.